app/views/components/address.scala.html

```diff
@@ -43,10 +43,11 @@
         label = messages("supplementary.address.postCode")
     )
 
-    @components.fields.field_autocomplete(
+    @components.fields.field_autocomplete_with_labelclass(
```

Comment:
Couldnt we just add an optional label class to the existing autocomplete?

Reply:
Thought of that.  Existing component has var-args as last param which precludes default arg value and most of the places that use that component (lots) don't use named arg lists.

Thought this was the simplest solution

Reply:
15 usages would need to be fixed, thats not _thaaaat_ many